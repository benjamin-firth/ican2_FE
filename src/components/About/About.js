import React from 'react';
import './About.scss';

const About = () => {

  return (
    <section className='about-container'>
      <h5>We are so glad you're here. Sign up or log in to start getting inspired</h5>
    </section>
  );
}

export default About;
