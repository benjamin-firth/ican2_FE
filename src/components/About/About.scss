@import '../../variables.scss';

.about-container {
  height: 100vw;
  background-color: $primarycolor;
  padding-top: 7vw;
  text-align: center;

  h5 {
    font-family: $secondaryfont;
    color: $secondarycolor;
    font-size: 2vw;
  }
}
