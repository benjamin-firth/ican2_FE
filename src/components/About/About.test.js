import React from 'react';
import { shallow } from 'enzyme';
import About from './About';

describe('About', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow( <About />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});