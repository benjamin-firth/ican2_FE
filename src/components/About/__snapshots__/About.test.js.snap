// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should match the snapshot 1`] = `
<section
  className="about-container"
>
  <h5>
    We are so glad you're here. Sign up or log in to start getting inspired.
  </h5>
</section>
`;
